src/util/__test/cycle-checker.util.test.ts

```diff
@@ -3,13 +3,25 @@ import * as path from 'path';
 import { CycleCheckerUtil } from '../cycle-checker.util';
 
 describe('CycleCheckerUtil', () => {
-    it('should detect a simple cycle', () => {
-        const graphCreator: GraphCreator = GraphCreator.builder()
-            .withRepoRoot(path.join(__dirname, './fixtures'))
-            .withDirectoryToCheck(path.join(__dirname, './fixtures'))
-            .build();
+    const tests: [string, string, boolean][]= [
+        ['simple-cycle', 'should detect a simple cycle', true],
+        ['no-cycle-simple', 'should detect no cycle', false]
+    ]
 
-        const graphWithCycle = graphCreator.createGraphForDir();
-        expect(CycleCheckerUtil.checkForCycles(graphWithCycle)).toThrow();
+    tests.forEach(([filePath, description, throws]) => {
```

Comment:
I get the idea behind it and it's enough for do our power, but generally for tests, I think avoiding these generalizations is preferable:
* They tend to be not very extensible (e.g. if we want another test where we check the output to the console)
* Really doesn't matter here, but with a big test-suite, being able to full-text-search a specific test-identifier ("should...") and immediately ending up at the point where the test is executed is quite valuable (this is still somewhat the case here but I guess you get the idea)
* DRY is a bit overrated anyway :P